### PR TITLE
EDU-630 Make all plugins work with width instead of maxWidth

### PR DESCRIPTION
**Pull request by ahelmberger**
Closes https://educandu.atlassian.net/browse/EDU-630

### CODE REVIEW

File: src/plugins/diagram-net/diagram-net.less [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
   margin: 0 auto;
   max-width: 100%;
 }
+
+.DiagramNetEditor {}
+
+.DiagramNetEditor-preview {
+  max-height: 80px;
+  max-width: 400px;
+  border: 1px solid #d9d9d9;
```
Comment (adrianaluchian) [inline src/plugins/diagram-net/diagram-net.less:14]: Could we use one of our defined border colors here instead of the custom HEX code?

**Review by adrianaluchian — APPROVED** [review]

**Comment by adrianaluchian** [comment]
Works super nicely 👍

File: src/plugins/image/image-editor.js [hunk 2]
```diff
@@ -359,13 +336,11 @@ function ImageEditor({ content, onContentChanged }) {
...
+          label={t('common:width')}
           {...formItemLayout}
-          validateStatus={smallImageSizeWarning ? 'warning' : null}
-          help={smallImageSizeWarning}
           >
-          <ObjectMaxWidthSlider value={maxWidth} onChange={handleMaxWidthValueChanged} />
+          <ObjectWidthSlider value={width} onChange={handleWidthValueChanged} />
         </Form.Item>
       </Form>
     </div>
```
[the remark this answers is not in the document]
Comment (ahelmberger) [reply src/plugins/image/image-editor.js:346]: Yeah :sunglasses:

File: src/plugins/diagram-net/diagram-net.less (hunk 1 above)
Comment (ahelmberger) [reply src/plugins/diagram-net/diagram-net.less]: Sure, this was actually `@edu-border-color-base` (probably by coincidence)

**Pull request merged**